### DISCUSSION TITLE
Segmentation fault when reading large material configuration file

### DISCUSSION THREAD

**Discussion by 123fgh123** [damask-multiphysics/DAMASK#49] [Q&A] [answered]
I founded that when the material configuration file is quite large, the simulation can’t run and lead to an error “Segmentation fault”：

"
<<<+-  config init  -+>>>

 reading material configuration

===================================================================================
=   BAD TERMINATION OF ONE OF YOUR APPLICATION PROCESSES
=   PID 3653756 RUNNING AT node932
=   EXIT CODE: 139
=   CLEANING UP REMAINING PROCESSES
=   YOU CAN IGNORE THE BELOW CLEANUP MESSAGES
===================================================================================
YOUR APPLICATION TERMINATED WITH THE EXIT STRING: Segmentation fault (signal 11)
This typically refers to a problem with your application.
Please see the FAQ page for debugging suggestions
"

I have attached my simulation files for further check.

Thank you.

[large_mat_file.zip](https://github.com/user-attachments/files/18082429/large_mat_file.zip)

**Comment by ToxFox88** [comment, marked as answer]
Try to add "ulimit -s unlimited" string before DAMASK running or include it to your *.sh file if you use it.

**Reply by 123fgh123** [reply]
I have tried your method and it works well. Thank you very much!
